elasticdl/python/master/k8s_worker_manager.py

```diff
@@ -84,6 +84,7 @@ def update_status(self, status):
         )
 
     def start_workers(self):
+        logger.info("Launching %d workers", self._num_workers)
```

Comment:
This is unnecessary since we are already printing out all the parsed arguments

Reply:
Good suggestion, removed.